### ISSUE TITLE
Registry as a Postgres table

### ISSUE THREAD

**Issue by kirill-zaitsev** [nossrannug/feast-postgres#4]
Hi! Following the README, I try to store the registry as a Postgres table:
```
registry:
    registry_store_type: feast_postgres.PostgreSQLRegistryStore
    path: feast_registry    # This will become the table name for the registry
    host: localhost
    port: 5432              # Optional, default is 5432
    database: mydb
    db_schema: dbschema.sql
    user: myuser
    password: mypassword
```

and `feast apply` gives me
`feast.errors.FeastClassImportError: Could not import RegistryStore 'PostgreSQLRegistryStore' from module 'feast_postgres'`

The feast packages are:
```
feast                           0.15.1
feast-postgres                  0.1.6
```

How can I store the registry as a Postgres table?

**Comment by nossrannug** [comment]
Hey @kirill-zaitsev,
The postgres registry is still a work in progress. I thought I had finished it and pushed the README, but haven't completed publishing to pypi. I expect to publish it later today or tomorrow.

**Comment by nossrannug** [comment]
Hey @kirill-zaitsev, I just published 0.2.0 (https://pypi.org/project/feast-postgres/). It has the postgres registry support.
Can you try it out and let me know if there are any issues please?

**Comment by kirill-zaitsev** [comment]
Thank you. It works with basic usage, I will let you know if things break in the scenarios where I plan to use it.
